local-interchain/ts/src/index.ts

```diff
@@ -0,0 +1,98 @@
+import { Coin, SigningStargateClient, SigningStargateClientOptions, GasPrice } from '@cosmjs/stargate';
+import { DirectSecp256k1HdWallet } from "@cosmjs/proto-signing";
+
+// cosmos1hj5fveer5cjtn4wd6wstzugjfdxzl0xpxvjjvr
+export const Mnemonic = "decorate bright ozone fork gallery riot bus exhaust worth way bone indoor calm squirrel merry zero scheme cotton until shop any excess stage laundry"
```

Comment:
we use this same mnemonic across all test in spawn, is known to be compromised :)